_posts/2022-08-02-leaving-facebook-meta-best-for-trino.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: "Why leaving Facebook/Meta was the best thing we could do for the Trino Community"
+author: "Martin Traverso, Dain Sundstrom, and David Phillips"
+excerpt_separator: <!--more-->
+---
+
+It might surprise some that our departure from Facebook was one of the simplest 
+decisions we’ve ever made. Many posts that discuss leaving a FAANG company focus
+on leaving some grand sum of money or prestige of working at the company. For 
+us, we were leaving the company where we had launched a project that we knew 
+would quickly outgrow the walls of Facebook, and solve a much larger set of 
+problems in the analytics domain. At the time we didn’t quite anticipate that 
+Presto, a distributed SQL query engine for big data analytics, would be adopted 
+around the globe by thousands of companies and an overwhelming number of 
+industries. We appreciate Facebook for serving as the launchpad that inspired 
+others to adopt Presto. Despite the harmonious beginnings, once the needs of the
+community and Facebook no longer aligned, we had to leave, but we’ll get to that
+part shortly.
+
+<!--more-->
+
+![|624x385](/assets/blog/leaving-facebook-meta-best-for-trino/original-gang.png)
+
+# People make up communities, not companies
+
+When we created Presto, it was clear to us that it needed to be open source.
+Presto started in 2012, just before the Facebook IPO. The culture was very
+conducive to starting an open source project. At that time, Facebook was working
+on Open Compute which ended up disrupting the hardware industry, and we wanted
+to do the same thing for the analytics industry with Presto. We lobbied for and
+gained approval from the VP of Infrastructure, Jay Parikh, and released 
+[Presto as an open source project](https://web.archive.org/web/20220203224702/https://www.computerworld.com/article/2485668/facebook-goes-open-source-with-query-engine-for-big-data.html). It’s something that we wanted to
```

Comment:
is there some better link?

Reply:
I want to use web archive since these websites are flakey and dissapear